river/drift/ddm.py

```diff
@@ -37,7 +36,7 @@ class DDM(DriftDetector):
 
     * if $p_i + s_i \geq p_{min} + 3 * s_{min}$ -> Change detected
 
-    **Input:** `value` must be a binary signal, where 1 indicates error.
+    **Input:** `x` must be a binary signal, where 1 indicates error.
```

Comment:
```suggestion
    The input `x` must be a binary signal, where `True` indicates an error.
```

Reply:
I left a comment about that in the main thread!